internal/graphql/schema/schema.graphql

```diff
@@ -241,6 +241,35 @@ type NameSpace {
 	namespace: String!
 }
 
+# ============== Patterns =================================
+
+type PatternPageResult {
+	page: Int!
+	page_size: Int!
+	total_count: Int!
+
+	patterns: [PatternResult]
+}
+
+type PatternResult {
+	id: ID!
+	name: String!
+	user_id: String!
+	location: PatternLocation!
+	pattern_file: String!
+	canSupport: Boolean!
+	errmsg: String
+	created_at: String
+	updated_at: String
+}
+
+type PatternLocation {
```

Comment:
Is "org" captured in here somewhere already?

Reply:
I don't think that we have `org` field even in the REST API response

Reply:
Ok, it must be captured under "host", I'm guessing.